docs/src/pages/components/about-the-lab/about-the-lab-de.md

```diff
@@ -20,20 +20,18 @@ Damit eine Komponente zum Kern (Core) übergehen kann, werden folgende Kriterien
 Installieren Sie das Paket innerhalb des Projektordners mit:
 
 ```sh
-// mit npm
 npm install @material-ui/lab
-
-// mit yarn
+```
```

Comment:
The changes to translation files are unnecessary since they'd be reverted immediately. Only change the english markdown files if required.

Reply:
Ok, I will change only english documents, but there will be a problem while copying the install blocks from other languages doc. Because yarn and npm install commands will be copied together.

![image](https://user-images.githubusercontent.com/39594189/128556168-3d468e1a-efee-4952-a29c-32a6a20c4225.png)